site/sass/components/_colors.scss

```diff
@@ -4,6 +4,25 @@
     #{$name}: #{$v};
   }
 
-  --black: #{map-get($google-colors, 'grey-900')};
-  --white: #ffffff;
+  $theming-colors: (
+    'primary-blue': map-get($google-colors, 'blue-650'),
```

Comment:
I was assuming that this would be changed to `blue-700` based on the color contrast feedback around CTAs. Is that not the case?